src/core/Database.cpp

```diff
@@ -34,9 +33,11 @@ Database::Database()
     , m_emitModified(false)
     , m_uuid(Uuid::random())
 {
-    m_data.cipher = KeePass2::CIPHER_AES;
+    m_data.cipher = KeePass2::CIPHER_CHACHA20;
```

Comment:
I suppose this means you'd also prefer that the old format be written where possible.

That will require some changes to KeePass2Writer.